<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailBot</class>
 <widget class="QMainWindow" name="MailBot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MailBot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_vendor_first_contact">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendor First Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_soft_pull">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Soft Pull</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reminder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Chronic/Damage Manual Pull</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-30</y>
      <width>811</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to the mail automationBot. Please choose the mail type from below:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buton_multiple_asins">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Delivery Retail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_no_stock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>No Stock</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
